common/lib/calc/setup.py

```diff
@@ -7,6 +7,6 @@
     install_requires=[
         "pyparsing==2.0.1",
         "numpy",
-        "scipy"
+        "scipy<0.18"
```

Comment:
I'm surprised we had been letting this version float.  Best thing here would be to pin this to a specific version of scipy, and also numpy.  Same for "chem".